ipynb/scripts/remove-blooms.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import qiime2 as q2
+from biom import load_table, Table
+
+bt = q2.Artifact.load('deblur-fmt-and-icu/table.qza').view(Table)
+
+blooms = []
+with open('newbloom.all.fna', 'r') as f:
```

Comment:
possible to add a comment noting the source? 

https://github.com/knightlab-analyses/bloom-analyses/blob/master/data/newbloom.all.fna